mod.ts

```diff
@@ -44,15 +44,16 @@ class Renderer extends Marked.Renderer {
     const html = Prism.highlight(code, grammar, language!);
     return `<div class="highlight highlight-source-${language} notranslate"><pre>${html}</pre></div>`;
   }
-
-  link(href: string, title: string, text: string) {
+  link(href: string, title: string | null, text: string) {
     if (href.startsWith("#")) {
       return `<a href="${href}" title="${title}">${text}</a>`;
```

Comment:
Looks like we also need the same fix here? (for in-page links)

Reply:
thanks!